osu.Game.Rulesets.Diva/Objects/Drawables/DrawableDivaHitObject.cs

```diff
@@ -124,14 +128,13 @@ public override void PlaySamples()
 
         protected override void CheckForResult(bool userTriggered, double timeOffset)
         {
-            Console.WriteLine(timeOffset);
-            if (Judged || base.Time.Current < HitObject.StartTime)
+            if (Judged)
             {
                 pressed = false;
                 return;
             }
-
-            if (!HitObject.HitWindows.CanBeHit(timeOffset))
+            
+            if (!HitObject.HitWindows.CanBeHit(timeOffset) && base.Time.Current > HitObject.StartTime)
```

Comment:
But just removing that then set this line to fail all notes right away, so I told it to wait until the note has PASSED before failing it.